settings/Validation/NoticePolicy.js

```diff
@@ -1,3 +1,12 @@
-export default function () {
-  return true;
+import FormValidator from './FormValidator';
+
+export default function (policy) {
+  const config = {
+    'name': {
```

Comment:
In this case it is not obvious, but key should be a string because it can be something like `'noticePolicies.period.name'` and for consistency in this file I suggest to use strings as object keys.